lib/middleman-cdn/commands.rb

```diff
@@ -22,23 +22,17 @@ def self.exit_on_failure?
         true
       end
 
-      desc "cdn:cdn_invalidate", "Invalidate your CloudFlare or CloudFront cache"
       def cdn_invalidate(*args)
         begin
-          if args.first && args.first.respond_to?(:filter)
-            options = args.first
-            files = args.drop(1)
+          options, files = if args.first && args.first.respond_to?(:filter)
+            [args.first, args.drop(1)]
           else
-            files = args
+            [Middleman::CDN::Extension.options, args]
```

Comment:
I thought a parallel assignment makes most sense here since we always want to set `options` and `files` based on whether the options are passed in as `args`.